tests/mock_tests/portsorch_ut.cpp

```diff
@@ -102,12 +152,19 @@ namespace portsorch_test
         pold_sai_queue_api = sai_queue_api;
         ut_sai_queue_api.get_queue_attribute = _ut_stub_sai_get_queue_attribute;
         sai_queue_api = &ut_sai_queue_api;
+
```

Comment:
Can we have this under _hook_sai_port_api()?

Reply:
queue/buffer mocking was not part of the PR. it is displayed here just because Vivek's PR which reverted this change has been merged. I'll remove this part accordingly.